src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java

```diff
@@ -41,12 +41,26 @@ public FindFullTextAction(BasePanel basePanel) {
 
     @Override
     public void init() throws Exception {
+        if (basePanel.getSelectedEntries().size() == 0) {
```

Comment:
Wouldn't be a better approach to disable the menu item (as it is already done for various other actions)?

If selection size() is == 0 at this point you can just throw an exception.